core/Makefile

```diff
@@ -280,25 +281,26 @@ build_firmware: templates build_cross ## build firmware with frozen modules
 		PYOPT="$(PYOPT)" BITCOIN_ONLY="$(BITCOIN_ONLY)" \
 		BOOTLOADER_QA="$(BOOTLOADER_QA)" BOOTLOADER_DEVEL="$(BOOTLOADER_DEVEL)" \
 		DISABLE_OPTIGA="$(DISABLE_OPTIGA)" THP="$(THP)" NEW_RENDERING="$(NEW_RENDERING)" \
-		$(FIRMWARE_BUILD_DIR)/firmware.bin
+		BENCHMARK="$(BENCHMARK)" $(FIRMWARE_BUILD_DIR)/firmware.bin
```

Comment:
note that there's now a common section `SCONS_OPTS` where this will go when you're rebasing